CONTRIBUTING.md

```diff
@@ -145,6 +146,11 @@ via email is also possible - contact maintainer about the details.
 Code submitted as raw source files (not patches), files attached to issue
 comments, forum posts, diffs in non-git format, etc will be promptly ignored.
 
+Any changes to README.md have to consider the project's latest stable release, 
+not the latest commit from the alpha version. New feature additions and removals
+related to changes from the alpha are added to the README.md as part of one of 
+the release steps, when we review all the documentation for release.
```

Comment:
Okay, but please rephrase/simplify, etc.

Reply:
@johnnovak where should I start?

Reply:
"The README.md should reflect the state of the `main` branch." 

Kerp it simple, clear, and use the least amount of words. Don't overexplain.